src/handbook/product/verticals.md

```diff
@@ -4,14 +4,74 @@ navTitle: Market Verticals
 
 # Product Verticals
```

Comment:
I think what you are describing are Product Categories. Verticals typically refer to different industries.